x-pack/plugins/observability/public/pages/slo_details/slo_details.tsx

```diff
@@ -53,6 +57,31 @@ export function SloDetailsPage() {
 
   useBreadcrumbs(getBreadcrumbs(basePath, slo));
 
+  useEffect(() => {
+    if (!slo) {
+      return;
+    }
+
+    return setScreenContext({
+      screenDescription: dedent(`
+        The user is looking at the detail page for the following SLO
+        
+        Name: ${slo.name}.
+        Id: ${slo.id}
```

Comment:
i guess we also need to slo.instanceId if it's not equal to *

Reply:
it's available in the metadata below

Reply:
What metadata are you referring to?

Reply:
I wonder if using the actual doc fields `slo.name`, `slo.id` etc, would help the LLM be able to match the data to parameters for functions when those parameters use the field name.

Reply:
the `data` prop - that's also sent over to the LLM

Reply:
re: your second question: yes, that's useful to the LLM, but also available in the `data` prop

Reply:
@dominiqueclarke sorry I spoke to soon, I have not actually verified whether field names are available in the metadata. Where are the field names used (outside of SOs)?